e2e/case6.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+set -o errexit -o nounset -o pipefail
+
+# Case 6: generate a coverage report
+bazel coverage //jest/tests:case6 --instrument_test_targets
```

Comment:
`--instrument_test_targets` doesn't seem to have any effect on the `COVERAGE_OUTPUT_FILE` var existing. I'm not not too sure what purpose it should serve here or if it's even needed.

Reply:
this flag only affects `process.env.COVERAGE_MANIFEST`. It'll only add more items to the coverage manifest.